src/WebJobs.Script/Workers/Http/CustomHandlerHttpOptions.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Http
+{
+    public sealed class CustomHandlerHttpOptions
+    {
+        /// <summary>
+        /// Gets or sets the default authorization level for custom handler HTTP worker routes.
+        /// </summary>
+        public AuthorizationLevel DefaultAuthorizationLevel { get; set; } = AuthorizationLevel.Function;
+
+        /// <summary>
+        /// Gets or sets route mapping for a HTTP worker.
+        /// </summary>
+        public IEnumerable<HttpWorkerRoute> Routes { get; set; }
```

Comment:
does anything prevent /admin from being overtaken? maybe that validation occurs elsewhere already. But it'd be good to maybe have a test to ensure it works.

Reply:
I validated that admin endpoint is not overtaken by this config. Even with route prefix. Will add a test for this as well.